src/services/completions.ts

```diff
@@ -668,6 +642,7 @@ namespace ts.Completions {
         uniqueNames: UniqueNameSet,
         target: ScriptTarget,
         entries: SortedArray<CompletionEntry>): void {
+        const entryNames = new Set(entries.map(e => e.name));

```

Comment:
GitHub isn’t showing me where this function gets called, but is this one redundant with `uniqueNames` too?

Reply:
Yep! I forgot to update that one, but it's the JS identifier-based completion path that I moved to the end of the function.
(So technically it's not required to add items to uniqueNames, but I'm going to, in order to avoid future surprises.)